bedrock/settings/base.py

```diff
@@ -1739,6 +1742,9 @@ def JINJA_CONFIG():
         'source_filenames': (
             'js/libs/modernizr.custom.shiv-load.js',
             'js/base/mozilla-input-placeholder.js',
+            'js/libs/jquery.waypoints.min.js',
```

Comment:
Have we tested this in prod-mode? I find that often minifying already minified code breaks things.